src/Microsoft.ML.DataView/DataViewType.cs

```diff
@@ -117,7 +117,7 @@ public static NumberDataViewType SByte
             get
             {
                 return _instSByte ??
-                    Interlocked.CompareExchange(ref _instSByte, new NumberDataViewType(typeof(sbyte), "I1"), null) ??
+                    Interlocked.CompareExchange(ref _instSByte, new NumberDataViewType(typeof(sbyte), "sbyte"), null) ??
```

Comment:
>sbyte [](start = 103, length = 5)

We are elsewhere using the .NET type names for these things, are we not? So should this be `SByte`, not `sbyte`